source/server/configuration_impl.cc

```diff
@@ -194,7 +194,9 @@ InitialImpl::InitialImpl(const envoy::config::bootstrap::v3::Bootstrap& bootstra
                          const Options& options)
     : enable_deprecated_v2_api_(options.bootstrapVersion() == 2u) {
   const auto& admin = bootstrap.admin();
-  admin_.access_log_path_ = admin.access_log_path();
+  admin_.access_log_file_info_ = Filesystem::FilePathAndType{
+      static_cast<Filesystem::DestinationType>(admin.access_log_destination()),
```

Comment:
unsafe static cast.

Reply:
fixed by adding a function in `config/utility.h` in 723ca8e578699979f11c35a93571d49c41f6b35c